docs/usage/edsl.rst

```diff
@@ -403,13 +412,13 @@ or invalid set of index variables. For example, in the code:
 
   .. group-tab:: C++
 
-    .. literalinclude:: ../codeexamples/writing_edsl.cc
+    .. literalinclude:: ../../plaidml/edsl/edsl_doc.cc
       :language: cpp
       :lines: 133-136
```

Comment:
Still using `:lines:`, is that intended?

Reply:
fixed